crates/nu-std/std/testing.nu

```diff
@@ -255,7 +258,7 @@ def run-tests-for-module [
                 ''
             }
         }
-        | par-each {|test|
+        | par-each  --threads $env.threads {|test|
```

Comment:
could we give the number of threads to `run-tests-for-module` as an argument here instead of relying on environment variables?

Reply:
I wanted to avoid doing that in case run-tests-for-module is ever reused elsewhere but I can change that if necessary

Reply:
ok done

Reply:
thanks @Yethal :pray:

Reply:
i think it's like using global variables which i tend to find unclear and with side effects :thinking: 

with Rust and Nushell in mind, i'd say it's great if we can avoid that as much as possible :relieved: 

and it avoids *messing up* the environment of the user with `$env.threads` :ok_hand: